tests/integration/test_storage_kafka/test.py

```diff
@@ -596,6 +630,19 @@ def test_kafka_formats(kafka_cluster):
                 cluster.schema_registry_port
             ),
             'supports_empty_value': True,
+        },
+        'Avro': {
```

Comment:
Just curios - if you will uncomment the Avro section which is commenter out - will it work either?

Reply:
Yes, it works fine.
Should I keep it commented/remove/make dedicated test/something else?

Reply:
just remove. Your option in clearer.

Reply:
Removed.